src/renderer/store/modules/utils.js

```diff
@@ -246,6 +246,46 @@ const actions = {
     return filenameNew
   },
 
+  /**
+   * This writes to the clipboard. If an error occurs during the copy,
+   * a toast with the error is shown. If the copy is successful and
+   * there is a success message, a toast with that message is shown.
+   * @param {string} content the content to be copied to the clipboard
+   * @param {string} messageOnSuccess the message to be displayed as a toast when the copy succeeds (optional)
+   * @param {string} messageOnError the message to be displayed as a toast when the copy fails (optional)
+   */
+  async copyToClipboard ({ dispatch }, { content, messageOnSuccess, messageOnError }) {
+    const locale = i18n._vm.locale
+    const translations = i18n._vm.messages[locale]
+    if (navigator.clipboard !== undefined && window.isSecureContext) {
+      try {
+        await navigator.clipboard.writeText(content)
+        if (messageOnSuccess !== undefined) {
+          dispatch('showToast', {
+            message: messageOnSuccess
+          })
+        }
+      } catch (error) {
+        if (messageOnError !== undefined) {
+          dispatch('showToast', {
+            message: `${messageOnError}: ${error}`,
+            time: 5000
+          })
+        } else {
+          dispatch('showToast', {
+            message: `${translations.Clipboard['Copy failed']}: ${error}`,
```

Comment:
Please use `i18n.t()` (works the same as `$t()` in vue files) here instead of needing to handle the translations yourself.

Reply:
I did not realize `$t()` was accessible. I had been trying to pass it through. The reason I switched to handling the translation manually is because that is how it is handled in the `downloadMedia` function, but this is so much cleaner.